.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
:eyes: I assume this was generated by husky and it is intentional. :)

Reply:
Yep!